src/adagrams.js

```diff
@@ -1,8 +1,75 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+const _ = require('lodash');
+class Adagrams {
+
+
+  static drawLetters() {
+    const letters = [];
+    const letter_freq = new Map([
+      ['A', 9], ['N', 6], ['B', 2], ['O', 8], ['C', 2], ['P', 2], ['D', 4], 
+      ['Q', 1], ['E', 12], ['R', 6], ['F', 2], ['S', 4], ['G', 3], ['T', 6], 
+      ['H', 2], ['U', 4], ['I', 9], ['V', 2], ['J',1], ['W', 2], ['K', 1], 
```

Comment:
Why not use an Object here? The syntactic sugar is a little more readable, and I don't see you making heavy use of anything that `Map` gives you.